src/pyodide/internal/python.js

```diff
@@ -3,6 +3,7 @@ import { createTarFS } from "pyodide-internal:tarfs";
 import { createMetadataFS } from "pyodide-internal:metadatafs";
 import { default as ArtifactBundler } from "pyodide-internal:artifacts";
 import { default as LOCKFILE } from "pyodide-internal:generated/pyodide-lock.json";
+import { default as internalJaeger } from "pyodide-internal:jaeger";
```

Comment:
Looks like we need to fill this in with a `DisabledInternalJaeger` in order to avoid breaking Python in workerd?

Reply:
Ah I see your checklist item for this.